README.md

```diff
@@ -54,7 +54,15 @@ Clone the repository
 
   - Re-fetch the project: `$ fvm flutter pub get`
 
-- The project uses the package name, the app name and the project name of the template if `PACKAGE_NAME`, `APP_NAME` and `PROJECT_NAME` aren't specified.
+- The fields detail:
+
+  | Field name   | Is mandatory | Rationale                                                                       |
+  | :----------- | :----------: | :------------------------------------------------------------------------------ |
+  | PACKAGE_NAME |     YES      | The application package name. The naming convention follows `com.your.package`  |
+  | PROJECT_NAME |     YES      | The application project name. The naming convention follows `your_project_name` |
+  | APP_NAME     |     YES      | The application name.                                                           |
+  | APP_VERSION  |      NO      | The app version that is set when initialize the project. Default is `0.1.0`     |
+  | BUILD_NUMBER |      NO      | The build number that is set when initialize the project. Default is `1`        |
```

Comment:
tbh I feel like we should change the command into lowercase instead of uppercase with cap-lock like this 😔  At least let's use `Yes` - `No` first. for the field name, we can come back with another PR later. You can check some other CLI tools like `docker` `aws` for references.

Reply:
is it possible to run 
`make run --package-name={com.your.package} ...` instead of `make run PACKAGE_NAME={.......}` @manh-t ? 🤔

Reply:
@sleepylee I'm looking for the argument formatting like this `--package-name` but it seems to not possible in `make` 🤔 
Instead, we can use `make run package-name=com.your.package ...` or `package_name`. I think we can go with `camel_case` 💡 How do you think?

Reply:
we can think about the package name later 🤔 I think there must be a way